CAT/strategy/BECAT_strategy.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
Please add the related paper link in your PR description. Also, update README for your PR.